### PR TITLE
Escape single quote in issuer DN

### PR DESCRIPTION
**Pull request by carbone-kid**
This hotfix fixes an error in certificate Path-Based Access Restrictions (PBAR) when the issuer DN contains single-quote (`'`) characters.

### CODE REVIEW

File: buildpack/nginx.py [hunk 1]
```diff
@@ -290,6 +290,7 @@ def get_access_restriction_locations():
                 # Escape special characters
                 issuer = issuer.replace(" ", "\\040")
                 issuer = issuer.replace(".", "\\.")
+                issuer = issuer.replace("'", "\\'")
```
Comment (markbastiaans) [inline buildpack/nginx.py:293]: This will actually become `\'` instead of `'` - is this the intention?

**Review by markbastiaans — CHANGES_REQUESTED** [review]
One code comment / question.

Since this should warrant a hotfix, be sure to:
- Base on `master`
- Add a description in the PR
- Bump the version
- Choose the "Squash and merge" option when merging

After merging, there will be a draft release in the Releases section. Publishing it will release this version into the wild.

**Comment by markbastiaans** [comment]
I've based this branch on `master` - we should not forget to rebase `develop` after releasing.

File: buildpack/nginx.py (hunk 1 above)
Comment (carbone-kid) [reply buildpack/nginx.py:293]: I beleave it should render `\'` in the nginx.conf and the check should work for `'`

Comment (markbastiaans) [reply buildpack/nginx.py:293]: Okay. Maybe it's good to create an integration test which verifies this. It's quite easy to spin up an application, apply a path restriction and do a HTTP request with an Issuer DN header set.

**Review by markbastiaans — APPROVED** [review]

**Pull request merged**